hoomd/md/NeighborListGPUBinned.cc

```diff
@@ -18,15 +18,9 @@ namespace py = pybind11;
 #endif
 
 NeighborListGPUBinned::NeighborListGPUBinned(std::shared_ptr<SystemDefinition> sysdef,
-                                             Scalar r_cut,
-                                             Scalar r_buff,
-                                             std::shared_ptr<CellList> cl)
-    : NeighborListGPU(sysdef, r_cut, r_buff), m_cl(cl), m_param(0)
+                                             Scalar r_buff)
+    : NeighborListGPU(sysdef, r_buff), m_cl(std::make_shared<CellList>(sysdef)), m_param(0)
```

Comment:
This should be on the GPU, yes?
```suggestion
    : NeighborListGPU(sysdef, r_buff), m_cl(std::make_shared<CellListGPU>(sysdef)), m_param(0)
```